salt/metalk8s/orchestrate/downgrade/precheck.sls

```diff
@@ -16,6 +18,13 @@ Correct saltenv "{{ saltenv }}" for downgrade to "{{ dest_version }}":
 
 {%- endif %}
 
+{%- if not defaults.downgrade_supported and not pillar.force_downgrade | default(False) %}
+
+Downgrade from current version is not supported:
```

Comment:
To me it's more `Downgrade from <current_version> to <dest_version> is not supported`
Maybe you can add the version here instead of "current version" :smile: